boards/stm32f429i-disc1/include/board.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2018 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for more
+ * details.
+ */
+
+/**
+ * @ingroup     boards_stm32f429i-disc1
+ * @{
+ *
+ * @file
+ * @brief       Board specific definitions for the STM32F429I-DISC1 evaluation board
+ *
+ * @author      Alexandre Abadie <alexandre.abadie@inria.fr>
+ */
+
+#ifndef BOARD_H
+#define BOARD_H
+
+#include "cpu.h"
+#include "periph_conf.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @name Macros for controlling the on-board LEDs.
+ * @{
+ */
+#define LED0_PIN            GPIO_PIN(PORT_G, 13)
+#define LED1_PIN            GPIO_PIN(PORT_G, 14)
+#define LED2_PIN            GPIO_PIN(PORT_B, 13)
+#define LED3_PIN            GPIO_PIN(PORT_C, 5)
```

Comment:
I'm wondering if LED2 & LED3 should be declared, there are related to USB OTG hardware right ?

Reply:
You are right, the datasheet is not calling them user LEDs by the way, I'll remove them.